### DISCUSSION TITLE
Your extended access has ended.

### DISCUSSION THREAD

**Discussion by iskandersierra** [community/community#39533] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I have a GitHub Copilot yearly plan, paid at 2022-08-22, and just today I received a message saing:

`Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended` with a button `Signup for GitHub Copilot`, that takes me to https://github.com/settings/billing, where I can see that I have the Add-on activated.

Is it a problem of cache on Visual Studio/VSCode? How can I refresh this status on my local tools? Where can I find any instruction on how to log-off/on from GitHub at both tools?

**Comment by beejaz** [comment]
I have the same issue but I pay monthly for three months already. And it only says on the Copilot section "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

**Comment by iamcjwilliams** [comment]
@iskandersierra Have you lost access?

**Comment by iskandersierra** [comment]
Hi everyone! Yes, I lost access in the first place. I just expected Copilot to pick the paid subscription from the moment the trial expired, but it didn't.
I just signed out and in again in VSCode and Visual Studio from the GitHub account, and restarted both of them (just for double measures), and it is working again.

In VSCode:
![image](https://user-images.githubusercontent.com/3475117/203269106-546532ca-4057-408b-bcd5-fa0df0526d62.png)

In Visual Studio:
![image](https://user-images.githubusercontent.com/3475117/203270179-a803e384-47ce-49d7-a7d4-9ea49adecf7c.png)
![image](https://user-images.githubusercontent.com/3475117/203270409-4316c8b5-e332-4752-bf4f-4db241266262.png)

I can close this issue for now, although it would be great if Copilot could pick the update without logging off of GitHub.

**Reply by ivanvanderbyl** [reply]
I have tried these steps, even uninstalling/reinstalling with a restart of VSCode in between signing out/in, but I'm still getting the error:

```
GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```

Any other suggestions?

**Reply by snaka** [reply]
I also tried Sign Out and restarting VS Code once to no avail.
However, after repeating the same steps a couple of times, Copilot became usable.

I'm not sure of the exact recovery procedure, but here's an FYI ...

**Reply by yubarajshrestha** [reply]
Not working for me either. It says 57 days of the trial still remain though but it does not work.

**Comment by davetapley** [comment]
I've just got similar:
> GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token"

Same thing signing out/in again in VSCode.

Output panel says:
```
[INFO] [auth] [2022-11-24T21:54:48.544Z] GitHub login failed
[ERROR] [default] [2022-11-24T21:54:48.546Z] GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token"
```

https://github.com/settings/billing shows:

<img width="789" alt="Screen Shot 2022-11-24 at 2 56 55 PM" src="https://user-images.githubusercontent.com/48232/203868066-282efe4b-765d-4d1c-ac68-ad65eea092f4.png">

**Comment by cozyGalvinism** [comment]
Got the same issue. What gives?

**Reply by cozyGalvinism** [reply]
Even logging out and logging back in, uninstalling and reinstalling both the extension and VSCode fail to work for me...

**Reply by cozyGalvinism** [reply]
I also don't want to sound too rude, but I kind of expected a product for $100 yearly to, well... not kill access to paying users.

**Reply by PabloLION** [reply]
I think the server is down at this moment.. Is there any status / health check? 
I can ping `vscode-auth.github.com` and `api.github.com` but not `copilot-proxy.githubusercontent.com`

**Reply by cozyGalvinism** [reply]
[There is](https://www.githubstatus.com/). But it seems there's nothing reported.

**Reply by PabloLION** [reply]
hmm weird. And they don't have a way to report issues in that page 😥

**Reply by cozyGalvinism** [reply]
If the Copilot server really is down, then I suppose someone is gonna get in mighty trouble...

**Reply by senecamanu** [reply]
omg we're all facing the same problem? i literally thought i'm the only one messing up my config/paying up.

**Reply by lonix** [reply]
i was sure i forgot to pay or something :)

**Reply by YassinHajaj** [reply]
Well... This is certainly not what I was expecting for a paid service.
Let's hope it gets back on its feet :)

**Reply by croquies** [reply]
Same!

**Reply by RyanZurrin** [reply]
Same, I had instances of it running fine yesterday and I have been paid member since the preview ended and had no problems until today. Now it is giving the same error across all my devices and across all my extensions in multiple IDE's and editors.

**Reply by manutzsong** [reply]
Had issue 2 hours ago.

**Reply by Benjamin-van-Heerden** [reply]
Also having this issue. Multiple editors across Windows and Linux.

**Comment by dev-fredericfox** [comment]
Same...

**Comment by Jfaler** [comment]
I got the same error

**Comment by vycos-zen** [comment]
i actively experiencing the same issue, after login off and back in vs code:

`[ERROR] [default] [2022-11-24T22:13:57.830Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."`

it stopped working within a hour ago in the US

code completion stopped, add-on disabled. I have still active monthly subscription paid up to date.

**Comment by PabloLION** [comment]
server down lalala

**Reply by dev-fredericfox** [reply]
Of course on Thanks Giving, classic move 😂

**Reply by PabloLION** [reply]
Maybe Copilot wants us to reflect and spend more time with family ❤️

**Reply by cozyGalvinism** [reply]
But there isn't Thanksgiving in Germany :sob: Besides, my family lives 400km away :laughing:

**Comment by trazyn** [comment]
Copilot server down???

**Comment by ai-slave** [comment]
**We're just checking how angry you are without AI writing your code instead of you.**

**Reply by PabloLION** [reply]
Nicely said by "AI-slave"

**Reply by cozyGalvinism** [reply]
Generally I would agree... but I can't exactly *not* use it after I've thrown $100 Microsoft's way. I'm a slave to my passion much like you are a slave to AI :(

**Reply by beauwilliams** [reply]
I don't need you AI, but I do need a persistent popup to signup for copilot which I already paid for to go away 😅

**Comment by beauwilliams** [comment]
I can't get copilot working...

**Comment by dmitrij-borchuk** [comment]
Same issue

**Comment by b14r** [comment]
same issue here.

**Comment by senecamanu** [comment]
still down eh? what's up with this

**Reply by dev-fredericfox** [reply]
I've got a feeling they are not aware that it is down yet.
https://www.githubstatus.com

Shows up as working. I tried to tweet at @githubstatus but I doubt they are going to see it on Thanks Giving.

**Reply by youssef-sobhy** [reply]
Yep it is

**Reply by senecamanu** [reply]
at this point im just praying really hard. copilot is my second brain, it's like losing your phone. damn.

**Reply by youssef-sobhy** [reply]
@senecamanu no man, you're the real brain!, just go back to the way it was until they fix whatever issues they're having :man_shrugging:

**Reply by dev-fredericfox** [reply]
Hope they can fix it without copilot 😂

**Reply by thorizer** [reply]
it's not a second brain but it's insane how time saving it is on repetitive front end tasks

**Reply by PabloLION** [reply]
Now I practice how to code without copilot. Maybe it's helpful for interview.

**Reply by senecamanu** [reply]
aint gonna lie, imma just wait for copilot to go back up bro 😂

**Reply by thedv91** [reply]
it's already working

**Comment by raphaelmansuy** [comment]
I face the same issue 😭

**Comment by vycos-zen** [comment]
tabnine can ba fair alternative, but not alike. at least for the time being.

**Comment by hashik4d** [comment]
Facing the same issue Tired to logout for github in VSCode and sign back in but NO luck.... @github-staff  any clue on who to reach out ?

**Comment by lee101** [comment]
I make an alternative cheaper API for OpenAI https://text-generator.io - i should really use my own product instead of paying Github for this... 
I'm just looking into faux pilot but running on my machine isn't really an option for doing ML work as I need the GPU, so hooking that up to my site would be great... that kind of begs the question why don't we just pay OpenAI directly for predictions? The OpenAI Edit API is still in free beta so you could really get a free Copilot out of that pretty easily

**Reply by lee101** [reply]
Basically extra auth step added by GitHub is making this down while OpenAI is up which is why we shouldn't design systems with too many middlemen like this

**Comment by Iscalej** [comment]
Same error. I purchased the annual subscription 👎

**Comment by iwangbowen** [comment]
Same here

**Comment by gin-melodic** [comment]
Same here...At first I thought it was a network problem.

**Comment by kyun** [comment]
My world is paused.

**Comment by luisheleon** [comment]
Same issue

**Comment by tuanle96** [comment]
same issue

**Comment by emanuelefavero** [comment]
same issue here in Italy. I've signed out and signed in again, reinstalled the copilot extension but copilot is still not working.

**Comment by mzavattaro** [comment]
Same issue in Australia, still down.

**Comment by JakeLaoyu** [comment]
Same issue

**Comment by Helfull** [comment]
What am i supposed to do, 3am in the night, if my AI programmer is down 😭 

I cant work anymore without it 

Please ETA till fix

**Comment by zhyu** [comment]
Same issue.

**Comment by zhzwz** [comment]
Same issue.

**Comment by ravhirizaldi** [comment]
It's happened to me today! My subs still active till next month

**Comment by Helfull** [comment]
![image](https://user-images.githubusercontent.com/720645/203886614-81d68f4d-2341-4bd9-b493-d425220dc05f.png)

**Reply by Helfull** [reply]
Yeah not really

**Comment by cihansahin** [comment]
Do I have to write the boring code myself now?

**Reply by edricgalentino** [reply]
yes you are hahaha, but same here

**Comment by wo1fsea** [comment]
Got the same issue...

**Comment by goncharenko** [comment]
Same issue

**Comment by chenchenfang** [comment]
same issue  .....

**Comment by i99dev** [comment]
Good time to learn Now, without co-pilot 😁 --> be Positive 

🤥 same issue

**Comment by mingo023** [comment]
same issue

**Comment by ivishalgandhi** [comment]
Same issue

**Comment by Coordinate-Cat** [comment]
hi, me too

**Comment by martin-ro** [comment]
same here

**Comment by dnapier** [comment]
Add another person to this growing list.

**Comment by osean-man** [comment]
It's down for me too, just wanted to chime in for Co-Pilots first downtime 🤣

**Comment by 0dayCTF** [comment]
Down for me as well, come back! I don't want to write code by hand, this is ridiculous lol

**Comment by DaemonStark** [comment]
Down for me as well!!🤔

**Comment by AndrianBalanescu** [comment]
Radio check ...

**Comment by edricgalentino** [comment]
owh it's down... i dont know that

**Comment by spaceman023** [comment]
Feature Request: Something that informs us what is actually going on when the server goes down without us having to search google for an answer.

**Reply by hassanzohdy** [reply]
The AI needs an AI to help

**Comment by yaocccc** [comment]
same here,

**Comment by dlopes7** [comment]
I am having the same issue the entire day today, signed out and back in multiple times, reinstalled the extension as well, nothing seems to work.

Very frustrating experience for a paid service.

**Comment by JohannLai** [comment]
```
GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```

the same here, with monthly plan

**Comment by beswift** [comment]
Same issue with vs code and pycharm.  

Honestly forgot how annoying it was to code without co-pilot.

**Reply by lulasz** [reply]
"We don't appreciate things until we lose them" 🤓

New marketing tactic? x)

**Comment by lccmanuel** [comment]
well same issue, here

**Comment by koukibuu3** [comment]
same here.

**Comment by HiepPP** [comment]
Same issue.

**Comment by morgangallant** [comment]
First time programming without Copilot in a few months — productivity has tanked, find myself pausing a lot waiting for completions 😂.

We're all in this together!

**Reply by RyanZurrin** [reply]
yeah, It does suck. It is like I know what I need to type but I have become totally lazy now and just want to press the tab key and have it all filled in automatically, lol.

**Comment by RyanZurrin** [comment]
I may have to switch to Tabnine after this experience.

**Reply by HiepPP** [reply]
what a coincidence, Tabnine have 50% discount today.

**Comment by tungv** [comment]
https://www.githubstatus.com/incidents/69flmdbxhtsk copilot server is down

**Comment by Abdullamhd** [comment]
Down for me too , tried all above steps but no lack

**Comment by yubarajshrestha** [comment]
Same for me too :(

**Comment by leizongmin** [comment]
Same for me too :(

**Comment by yogurt7771** [comment]
Same issue

**Comment by mechanicles** [comment]
Same issue :(

**Comment by FlatMapIO** [comment]
We found an issue that is causing some Copilot users to incorrectly receive a response indicating their subscription has ended. We are continuing to work on mitigation.

> Posted 1 minute ago. Nov 25, 2022 - 05:48 UTC

**Comment by shd101wyy** [comment]
same issue :(

**Comment by davecheney** [comment, marked as answer]
Hello. I'm sorry you had difficulty accessing Copilot, please accept my apologies on behalf of the Copilot team. This was caused by a bad deploy which affected a small percentage of Copilot users. The change has been rolled back and you should be able to use Copilot again.

**Reply by Hybeecodes** [reply]
I still am unable to access copilot. Still getting the error

**Reply by yogurt7771** [reply]
I can access copilot now

**Reply by Hybeecodes** [reply]
I can also access copilot now

**Reply by Benjamin-van-Heerden** [reply]
Still unable to access. Getting a "Thank you for participating in the GitHub Copilot Technical Preview" message (despite paying for the service) and Copilot not working across multiple IDEs  on Windows and Linux

**Reply by ajheunis** [reply]
I have this issue as well. I tried logging out and in, but the issue remains.

**Reply by skyne98** [reply]
Same issue remains.

**Reply by dontWatchMeCode** [reply]
Stopped working ~10 min ago,
well, let's see if I can still code ¯\_(ツ)_/¯

**Reply by thedv91** [reply]
Same with @ajheunis

**Reply by HabunoGD1809** [reply]
I have the same problem, I am a member "GitHub Student Developer Pack
"and I had my COPILOT working 100%. But it's been less than an hour since this has been causing problems.

![image](https://user-images.githubusercontent.com/68973717/204065948-f8857b5b-bbb4-4d97-99ae-c24c0feda7fd.png)

**Reply by jeremiahsherrill** [reply]
not answered

**Reply by titusdecali** [reply]
I paid for a year and have tried signing out and in (4 times), restarting VScode. Uninstall Copilot sign out, sign in, reinstall...
Nothing works. 

![SHOT 10 14 50@2x](https://user-images.githubusercontent.com/4532534/204066960-63de616f-b55c-41c0-beae-881d94caba72.jpg)


![SHOT 10 12 51@2x](https://user-images.githubusercontent.com/4532534/204066943-bd9965ac-aa48-4e42-9c44-be4c1fa745c6.jpg)

**Reply by RobxSoft** [reply]
Also having the same issue, I am a member "GitHub Student Developer Pack". But I can't use copilot at the moment.

**Comment by haoranyu** [comment]
same issue :(

**Reply by Hybeecodes** [reply]
Same here

**Comment by gin-melodic** [comment]
This incident has been resolved.
https://www.githubstatus.com/incidents/69flmdbxhtsk

**Reply by titusdecali** [reply]
not resolved

**Comment by pfcodes** [comment]
This outage made me realize how dependent i am on this service lol

**Reply by vycos-zen** [reply]
same here, i felt my half is missing already. but it is actually a good thing, it made me think more about what i write.

**Comment by Coordinate-Cat** [comment]
Resolved! Now the work will start! thx:)

**Comment by justkiddinglesha** [comment]
все еще выбивает такуб ошибку, как можно починить?
так же при попытке нажать на странице github-copilot - enable copilot выдает ошибку:

Cannot proceed with payment
We are still investigating the issue with your GitHub account, and we will update you as soon as possible. Thank you for your patience during this process. We are sorry for any inconvenience.

**Comment by lijo-jose** [comment]
Facing the issue again
`[ERROR] [default] [2022-11-25T09:16:04.692Z] GitHub Copilot could not connect to server. Extension activation failed: "Timed out waiting for authentication provider to register"`
please help

**Comment by Arbust908** [comment]
Looks like it's now working!

**Comment by ndevvy** [comment]
In VS Code I'm still seeing a popup: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended"; suggestions aren't populating.  My Copilot subscription is active.

**Reply by skyne98** [reply]
Looks like they either haven't actually fixed the deployment issue or it's rolling out slowly.

**Reply by dontWatchMeCode** [reply]
i blame serverless, or kubernetes
actually probably both

**Reply by titusdecali** [reply]
That makes sense. I live in Asia, so maybe that explains why mine is still broken.

**Comment by dzonerzy** [comment]
Same Issue here

**Comment by beswift** [comment]
This was working again for most of the day, but just randomly stopped again a few min ago.

**Comment by dnapier** [comment]
Same experience as the last few comments. Working all day until just now.

**Comment by RyanZurrin** [comment]
Same issue, as everyone. It was working for most of the day and then all of a sudden it stopped and started behaving similar to yesterday’s issue.

**Comment by taesiri** [comment]
I am having the same issue.

**Comment by jeremiahsherrill** [comment]
I don't really use it all that much, but was working on some ruby couldn't remember a particular function and poof I am getting the same error.

**Comment by rohanmuplara** [comment]
same error

**Comment by zhaoxiongfei** [comment]
same issue here.
![image](https://user-images.githubusercontent.com/1306392/204066988-65919f0c-729b-4b7e-ae98-29ccb91c0d99.png)
![image](https://user-images.githubusercontent.com/1306392/204066937-7e1f0d9c-87c6-4c0e-b770-4d5d9a39c028.png)
I've tried logging into github again,
Tried reinstalling the github copliot,
Tried reinstalling vscode,
Tried deleting vscode cache ~/Library/ApplicationSupport/Code
Tried sign in, sign out of vscode repeatedly
Tried everything I can think of

**Comment by dnapier** [comment]
Working now but my IDE is trolling me.

![image](https://user-images.githubusercontent.com/19673339/204067198-b1bec2e5-1627-4a93-b554-549a2975e83a.png)

**Comment by albertkun** [comment]
working now!

**Comment by zhaoxiongfei** [comment]
working now!
